design-system/materials/internals/definition.yaml

```diff
@@ -264,6 +264,43 @@ choiceGroupsByTheme:
         break-point-giantdesktop: 1680px
         break-point-jumbodesktop: 1920px
 
+  recolouring:
+    colors:
+      label: Colors
+      prefix: color
+      description: All colors in the system
+      choices:
+        color-primary: 'hsl(240, 64%, 58%)'
+        color-primary-20: 'hsl(240, 66%, 19%)'
+        color-primary-25: 'hsl(240, 46%, 48%)'
+        color-primary-30: 'hsl(240, 46%, 53%)'
+        color-primary-40: 'hsl(240, 100%, 67%)'
+        color-primary-85: 'hsl(244, 100%, 84%)'
+        color-primary-90: 'hsl(243, 100%, 93%)'
+        color-primary-95: 'hsl(244, 100%, 97%)'
+        color-turquoise-20: 'hsl(180, 87%, 20%)'
+        color-turquoise-35: 'hsl(180, 70%, 35%)'
+        color-turquoise-50: 'hsl(180, 50%, 50%)'
+        color-turquoise-70: 'hsl(180, 60%, 70%)'
+        color-turquoise-85: 'hsl(180, 70%, 85%)'
+        color-turquoise-90: 'hsl(180, 80%, 90%)'
+        color-success: 'hsl(152, 77%, 39%)'
+        color-success-25: 'hsl(155, 84%, 20%)'
+        color-success-40: 'hsl(155, 90%, 24%)'
+        color-success-85: 'hsl(144, 69%, 80%)'
+        color-success-95: 'hsl(141, 76%, 92%)'
+        color-warning: 'hsl(35, 90%, 45%)'
+        color-warning-25: 'hsl(33, 83%, 25%)'
+        color-warning-40: 'hsl(33, 80%, 34%)'
+        color-warning-60: 'hsl(35, 90%, 55%)'
+        color-warning-85: 'hsl(33, 90%, 80%)'
+        color-warning-95: 'hsl(45, 100%, 92%)'
+        color-error: 'hsl(3, 65%, 58%)'
+        color-error-25: 'hsl(4, 69%, 37%)'
+        color-error-40: 'hsl(3, 60%, 46%)'
+        color-error-85: 'hsl(1, 55%, 74%)'
+        color-error-95: 'hsl(349, 66%, 92%)'
```

Comment:
@FilPob As discussed with Carlos, I added the new colors so hope you don't mind me asking for another review 🙂
(especially since there were a few cases when a color inspected in Figma dev mode was different than in the color square label)